static_table.go

```diff
@@ -101,3 +101,133 @@ var staticTableEntries = [...]HeaderField{
 	{Name: "x-frame-options", Value: "deny"},
 	{Name: "x-frame-options", Value: "sameorigin"},
 }
+
+type indexAndValues struct {
+	idx    int
+	values map[string]int
+}
+
+// A map of the header names from the static table to their index in the table.
+// This is used by the encoder to quickly find if a header is in the static table
+// and what value should be used to encode it.
+// There's a second level of mapping for the headers that have some predefined
+// values in the static table.
+var encoderMap = map[string]indexAndValues{
+	":authority":          indexAndValues{0, map[string]int{}},
```

Comment:
Why not use nil for values that don’t have a value?

Reply:
I just forgot this was possible in Go :)